pkg/algorithm/nodeselect/v1alpha1/select_node.go

```diff
@@ -186,6 +159,45 @@ func (ac *Config) selectNode(nodeBlockDeviceMap map[string]*blockDeviceList) *no
 	// minRequiredDiskCount will hold the required number of disk that should be selected from a qualified
 	// node for specific pool type
 	minRequiredDiskCount := blockdevice.DefaultDiskCount[ac.poolType()]
+
+	if ProvisioningType(ac.Spc) == ProvisioningTypeAuto {
+		// List all blockdeviceclaims related to spc
+		// NOTE: This list contains blockdevices on which csps' are already
+		// created filter and use it
+		bdcList, err := bdc.NewKubeClient().
+			WithNamespace(ac.Namespace).
+			List(metav1.ListOptions{LabelSelector: string(apis.StoragePoolClaimCPK) + "=" + ac.Spc.Name})
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to select node and blockdevices")
+		}
+		customBDCList := bdc.BlockDeviceClaimList{
```

Comment:
What is `customBDCList` ? Either we can make the var name more intuitive or put some comment.

Reply:
I will update with comments

Reply:
Done